apps/frontend/src/libs/components/icon/icon.tsx

```diff
@@ -0,0 +1,17 @@
+import { iconNameToSvg } from "./libs/maps/maps.js";
+import { type IconName } from "./types/types.js";
+
+type Properties = {
+	color?: string | undefined;
```

Comment:
Do we need explicitly write type as `string | undefined`? maybe better use just `string` as you do in `Input` component?

Reply:
Yes, here I can leave just `string`, but in this case I should also redo icon-button `using color={iconColor ?? ""}`

Reply:
I see it more as a matter of preference, so I'm not sure if I should redo it.

Reply:
In this case, leave color type as it is